src/models/exerciseDao.js

```diff
@@ -0,0 +1,31 @@
+const { AppDataSource } = require("./dataSource");
+
+const getRandomExercises = async (subscriptionState) => {
+  const limitedContentsQuery = !subscriptionState
+    ? `WHERE exercises.is_premium = false`
+    : ``;
+  const exercises = await AppDataSource.query(`
+    SELECT
+      id AS exerciseId,
+      thumbnail_url AS thumbnailURL,
+      name,
+      is_premium AS isPremium
+    From
+      exercises
+    ${limitedContentsQuery}
+    ORDER BY RAND()
+    LIMIT 5
+    ;
+  `);
+  return exercises;
+};
+
```

Comment:
```suggestion

const getRandomExercises = async (subscriptionState, limit = 5) => {
  const limitedContentsQuery = !subscriptionState
    ? `WHERE exercises.is_premium = false`
    : ``;
  const exercises = await AppDataSource.query(`
    SELECT
      id AS exerciseId,
      thumbnail_url AS thumbnailURL,
      name,
      is_premium AS isPremium
    From
      exercises
    ${limitedContentsQuery}
    ORDER BY RAND()
    LIMIT ?
    ;
  `, [limit]);
  return exercises;
};

```
선택하려는 데이터의 수 또한 언제든지 변경할 수 있도록 변수화 하는 것이 보다 확장성 있는 Dao를 구성할 수 있는 방법입니다. limit이 없을 때는 5로 기본값이 설정될 수 있도록 할 수 있습니다 :)